### PR TITLE
[Workspace] Disable copy all button when there is no saved objects in assets page 

### PR DESCRIPTION
**Pull request by yubonluo**
### Description

Page crashed after click "Copy assets to ..." button in workspace assets page when assets table is empty
image

image (5)

## Screenshot

image

## Changelog

- fix: Disable cope all button when there is no saved objects in assets page 
### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by SuZhou-Joe — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
It seems we only disable the button instead of fixing the root cause. Can we fix the place giving the error?

File: src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_result_flyout.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ export class DuplicateResultFlyout extends React.Component<DuplicateResultFlyout
         {copiedCount && (
           <EuiFlexItem grow={false}>
             <EuiTitle size="xs">
-              <h4 className="savedObjectsManagementImportSummary__copiedCountCount">
+              <h4 className="savedObjectsManagementImportSummary__createdCount">
```
Comment (wanglam) [inline src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_result_flyout.tsx:94]: Nit: This change does not seem related to the copy issue, was it made intentionally?

**Review by wanglam — APPROVED** [review]
Great fix. After testing, it looks good on my local environment.

**Comment by yubonluo** [comment]
> It seems we only disable the button instead of fixing the root cause. Can we fix the place giving the error?

Sure, the root cause is when selectedObjectCount is 0, we can not get the `objectName: allSelectedObjects[0].meta.title,`. After updating the code, it should be `objectName: allSelectedObjects[0]?.meta.title,`

**Review by Hailong-am — APPROVED** [review]
